test/e2e/reconfigure_test.go

```diff
@@ -28,14 +27,13 @@ func reconfigurationTest(t *testing.T) {
 	require.NoError(t, err)
 
 	// Remove the version annotation of one of each type of node
-	// Forward slash within a path is escaped as '~1'
-	escapedVersionAnnotation := strings.Replace(nc.VersionAnnotation, "/", "~1", -1)
-	patchData := fmt.Sprintf("[{\"op\": \"remove\", \"path\": \"/metadata/annotations/%s\"}]", escapedVersionAnnotation)
+	patchData, err := annotations.GenerateRemovePatch([]string{nc.VersionAnnotation})
```

Comment:
If you want to use this in test code, please add a unit test for the function

Reply:
Added (in other commit).